core/store/src/flat_state.rs

```diff
@@ -563,10 +563,10 @@ impl Into<i64> for &FlatStorageStateStatus {
     /// Cast inside enum does not work because it is not fieldless.
     fn into(self) -> i64 {
```

Comment:
is there a reason to use `i64` instead of `u64` here? do we want to use negative numbers at some point?

Reply:
ah, got it, thanks!